NuGet.config

```diff
@@ -3,6 +3,10 @@
   <packageSources>
     <add key="AspNetCore" value="https://dotnet.myget.org/F/aspnetcore-ci-dev/api/v3/index.json" />
     <add key="AspNetCoreTools" value="https://dotnet.myget.org/F/aspnetcore-tools/api/v3/index.json" />
+    <!-- Workaround for missing package on our feed -->
+    <add key="DotNetCore" value="https://dotnet.myget.org/F/dotnet-core/api/v3/index.json" />
```

Comment:
The System.Numerics.Vector that we spoke about yesterday. https://github.com/aspnet/CoreCLR/pull/175. Did this get fixed?